### ISSUE TITLE
Gives no hints at all

### ISSUE THREAD

**Issue by hacker112** [globexdesigns/brackets-jscs#14]
I tried to create a simple .jscs.json file and added it to my projects root-dir:
```json
{
    "preset": "jquery",
    "excludeFiles": ["node_modules/**"]
}
```
The jscs command line tool gives me errors. But Brackets gives me no hints at all. 
Other Brackets hinting extensions works fine like jsonhint and jshint.
I am using Brackets 1.1.

Any idea of why it is not working?

**Comment by globexdesigns** [comment]
The example .jscsrc file you pasted doesn't have any rules defined. The extension does not enable any rules by default. Try something like this:

```
{
	"excludeFiles": [
		"node_modules"
	],

	"validateJSDoc": {
		"checkParamNames": true,
		"checkRedundantParams": true,
		"requireParamTypes": true
	},

	"requireSpaceAfterKeywords": [
		"if",
		"else",
		"for",
		"while",
		"do",
		"switch",
		"return",
		"try",
		"catch",
		"function"
    ],

	"requireSpacesInFunctionExpression": {
		"beforeOpeningCurlyBrace": true
	},

	"disallowTrailingWhitespace": true,

	"disallowSpaceAfterObjectKeys": true,
	"disallowSpacesInsideObjectBrackets": true,
	"disallowSpacesInsideArrayBrackets": true,
	"disallowSpacesInsideParentheses": true,
	"disallowSpaceBeforeBinaryOperators": [
		"!",
		","
	],

	"requireSpaceBeforeBinaryOperators": [
		"=",
		"+",
		"-",
		"/",
		"*",
		"==",
		"===",
		"!=",
		"!=="
	],

	"requireSpaceAfterBinaryOperators": [
		":",
		"=",
		",",
		"+",
		"-",
		"/",
		"*",
		"==",
		"===",
		"!=",
		"!=="
	],

	"disallowSpaceAfterPrefixUnaryOperators": ["++", "--", "+", "~"],
	"disallowSpaceBeforePostfixUnaryOperators": ["++", "--"],

	"disallowKeywords": ["with"],
	"disallowKeywordsOnNewLine": ["else"]
}

```

**Comment by hacker112** [comment]
Thanks!
First it did not help at all. Still nothing. 
Then I remembered that I changed Brackets settings after upgrading to Brackets 1.1 a while back.
They changed how linters where used: all lint tools where used, which mean that even the built in jslint was used which gave me a lot of errors. So what I did then and forgot was:
```json
    "language": {
        "javascript": {
            "linting.prefer": [
                "JSHint"
            ],
            "linting.usePreferredOnly": true
        }
    },
```

So now I changed it to: 
```json
    "language": {
        "javascript": {
            "linting.prefer": [
                "JSHint","JSCS"
            ],
            "linting.usePreferredOnly": true
        }
    },
```

And it worked like a charm even my first config worked (the preset includes rules)!
So I would suggest that you add info in the readme about updating the Brackets preferences if someone else has added these preferences (and forgot that they did :P) 
Docs on the linting.prefer
https://github.com/adobe/brackets/wiki/How-to-Use-Brackets
